apps/preview/next/pages/showcases/NavbarTop/NavbarTop.tsx

```diff
@@ -26,40 +38,92 @@ export default function TopNav() {
     },
   ];
 
+  const search = (event: React.FormEvent<HTMLFormElement>, searchValue: string) => {
+    event.preventDefault();
+    alert(`Successfully found 10 results for ${searchValue}`);
+  };
+
   return (
-    <div className="w-full h-full bg-neutral-50">
-      <header className="flex justify-center w-full bg-white border-b h-14 md:h-20 border-neutral-200">
-        <div className="flex items-center flex-row flex-nowrap justify-start h-full max-w-[1536px] w-full px-4 md:px-10">
-          <a
-            href="/"
-            aria-label="SF Homepage"
-            className="inline-block mr-2 md:mr-10 focus-visible:outline focus-visible:outline-offset focus-visible:rounded-sm"
-          >
-            <picture>
-              <source srcSet={brandLogo.src} media="(min-width: 1024px)" />
-              <img src={brandLogoSign.src} alt="Sf Logo" className="w-8 h-8 lg:w-[12.5rem] lg:h-[1.75rem]" />
-            </picture>
-          </a>
-          <SfButton className="block mr-auto" type="button" slotSuffix={<SfIconExpandMore />} variant="tertiary">
-            <span className="hidden md:inline-flex">Browse products</span>
-          </SfButton>
-          <nav className="flex flex-row flex-nowrap">
+    <header className="flex justify-center w-full py-2 px-4 lg:py-5 lg:px-6 bg-white border-b border-neutral-200">
+      <div className="flex flex-wrap lg:flex-nowrap items-center flex-row justify-start h-full max-w-[1536px] w-full">
+        <a
+          href="/"
+          aria-label="SF Homepage"
+          className="inline-block mr-4 focus-visible:outline focus-visible:outline-offset focus-visible:rounded-sm shrink-0"
+        >
+          <picture>
+            <source srcSet={brandLogo.src} media="(min-width: 768px)" />
+            <img
+              src={brandLogoSign.src}
+              alt="Sf Logo"
+              className="w-8 h-8 md:h-6 md:w-[176px] lg:w-[12.5rem] lg:h-[1.75rem]"
+            />
+          </picture>
+        </a>
+        <SfButton
+          aria-label="Open categories"
+          className="lg:hidden order-first lg:order-1 mr-4"
+          square
+          variant="tertiary"
+        >
+          <SfIconMenu />
+        </SfButton>
+        <SfButton
+          className="hidden lg:flex lg:mr-4"
+          type="button"
+          variant="tertiary"
+          slotSuffix={<SfIconExpandMore className="hidden lg:block" />}
+        >
+          <span className="hidden lg:flex whitespace-nowrap">Browse products</span>
+        </SfButton>
+        <form
+          role="search"
+          className="flex flex-[100%] order-last lg:order-3 mt-2 lg:mt-0 pb-2 lg:pb-0"
+          onSubmit={(event) => search(event, inputValue)}
```

Comment:
Then here it's just 
```suggestion
          onSubmit={search}
```

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2638/commits/eae1bae35fa1ae13223452be0af6064f452d573a